Source/swiftlint/LintCommand.swift

```diff
@@ -41,47 +40,63 @@ struct LintCommand: CommandType {
         }
     }
 
-    private func lint(path: String,
-        configuration: Configuration, strict: Bool) -> Result<(), CommandantError<()>> {
+    private func lint(path: String, configuration: Configuration, strict: Bool) ->
+        Result<(), CommandantError<()>> {
         let filesToLint = (configuration.included.count == 0 ? filesToLintAtPath(path) : [])
             .filter({ !configuration.excluded.flatMap(filesToLintAtPath).contains($0) }) +
             configuration.included.flatMap(filesToLintAtPath)
-        if filesToLint.count > 0 {
-            if path.isEmpty {
-                print("Linting Swift files in current working directory")
-            } else {
-                print("Linting Swift files at path \(path)")
-            }
-            var numberOfViolations = 0, numberOfSeriousViolations = 0
-            for (index, path) in filesToLint.enumerate() {
+        if path.isEmpty {
+            fputs("Linting Swift files in current working directory\n", stderr)
+        } else {
+            fputs("Linting Swift files at path \(path)\n", stderr)
+        }
+        let files = filesToLint.flatMap(File.init)
+        if files.count > 0 {
+            return lint(files, configuration: configuration, strict: strict)
+        }
+        return .Failure(CommandantError<()>.UsageError(description: "No lintable files found at" +
+            " path \(path)"))
+    }
+
+    private func lint(files: [File], configuration: Configuration, strict: Bool) ->
+        Result<(), CommandantError<()>> {
+        var violations = [StyleViolation]()
+        var reporter: Reporter.Type = XcodeReporter.self
+        for (index, file) in files.enumerate() {
+            if let path = file.path {
                 let filename = (path as NSString).lastPathComponent
-                print("Linting '\(filename)' (\(index + 1)/\(filesToLint.count))")
-                let file = File(path: path)!
-                for violation in Linter(file: file, configuration: configuration).styleViolations {
-                    fputs("\(violation)\n", stderr)
-                    numberOfViolations++
-                    if violation.severity == .Error {
-                        numberOfSeriousViolations++
-                    }
-                }
+                fputs("Linting '\(filename)' (\(index + 1)/\(files.count))\n", stderr)
```

Comment:
I think you have this flipped, since the opposite is now true: all informational logs can now be silenced by `2> /dev/null`, so you can now pipe the result of swiftlint like this: `swiftlint > violations.json` or `swiftlint > violations.csv`.